packages/headless/src/features/triggers/trigger-analytics-actions.ts

```diff
@@ -45,3 +45,23 @@ export const logTriggerRedirect = makeAnalyticsAction(
     return;
   }
 );
+
+/**
+ * Log trigger execute
+ */
+export const logTriggerExecute = makeAnalyticsAction(
+  'analytics/trigger/execute',
+  AnalyticsType.Search,
+  (client, state) => {
+    if (state.triggers?.execute) {
+      return client.logTriggerExecute({
+        executed:
+          'function name: ' +
+          state.triggers.execute.functionName +
+          ' params: ' +
+          state.triggers.execute.params,
+      });
```

Comment:
I thought it was weird that the log action only accepted a string which I assume should be the function name. I thought it would be more informative to provide both the function name and the parameters so I opted for this format but I'm not sure if it's adequate for those handling the analytics.

Reply:
https://github.com/coveo/search-ui/blob/master/src/ui/Triggers/Triggers.ts#L141